### PR TITLE
IDVA6-659 remove auth check on adding new authorised person

### PR DESCRIPTION
**Pull request by mmodzelewski-ch**
Link to Jira:
https://companieshouse.atlassian.net/browse/IDVA6-659

Changes:
- Auth check removed from adding a new authorised person journey
- Issues flagged by SonarLint fixed

### CODE REVIEW

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

File: test/src/routers/controllers/addPresenterController.test.ts [hunk 1]
```diff
@@ -15,7 +15,7 @@ const router = supertest(app);
 const en = require("../../../../src/locales/en/translation/add-presenter.json");
 const cy = require("../../../../src/locales/cy/translation/add-presenter.json");
 const companyNumber = "12345678";
-const urlwithCompNum = "/your-companies/company/:companyNumber/add-presenter";
+const urlwithCompNum = "/your-companies/add-presenter/:companyNumber";
```
Comment (kkonuganti-ch) [inline test/src/routers/controllers/addPresenterController.test.ts:18]: what url is add-presenter?

Comment (kkonuganti-ch) [reply test/src/routers/controllers/addPresenterController.test.ts:18]: is it our page?

Comment (mmodzelewski-ch) [reply test/src/routers/controllers/addPresenterController.test.ts:18]: We usually follow the URLs in the prototype so Brendan grabbed it from there. It's the add new authorised person URL. I have no idea why it's named this way. Usually the URLs are fairly sensible. Here is a link to the prototype https://find-update-company-info.herokuapp.com/v6/add-presenter

**Pull request merged**